src/content/quickstart/gke/_index.md

```diff
@@ -0,0 +1,156 @@
+---
+date: 2020-11-13T17:55:18+01:00
+title: "Google Kubernetes Engine (GCP)"
+weight: 20
+---
+
+This quickstart guide covers the necessary steps to deploy two Google Kubernetes Engine (GKE) clusters on Google Cloud Platform (GCP) using `gcloud`. The clusters are named `cluster-a` and `cluster-b`. 
+
+Once the GKE clusters are deployed, we deploy Submariner with Service Discovery to interconnect the two clusters. Note that this guide focuses on Submariner deployment on clusters with non-overlapping Pod and Service CIDRs. For connecting clusters with overlapping CIDRs, please refer to the [Submariner with Globalnet guide](./globalnet/), which was not tested for this tutorial.
+
+{{% notice info %}}
+The guide assumes that you have a working GCP account and the `gcloud` binary installed and configured. To get the binary, please visit [Google's website](https://cloud.google.com/sdk/docs/install).
+{{% /notice %}}
+
+
+## Cluster creation
+Create two identical K8s clusters on GKE. For this demo, the following configuration was used:
+``` bash
+gcloud container clusters create "cluster-a/b" \
+    --zone "<zone-id>" \
+    --cluster-version "1.17.13-gke.1400" \
+    --username "admin" \
+    --machine-type "g1-small" \
+    --image-type "UBUNTU" \
```

Comment:
nit: Is it necessary to use the ubuntu type of image?

Reply:
to be honest, I'm not sure. I wanted to provide the exact config I got it working with, because I don't know what could case other problems ... hm

Reply:
:+1: I can try it later and if not necessary we can remove it on a separate PR.